Correction in the energy control point assignment and readme update (#37)
SebastiaanBreedveld/TROTS: TOPAS_simulation/README.md

Reviewer: ```suggestion
   a. Operation: Export------Output type: Labelmap------Output node: Export to new labelmap. This will generate an NRRD file.
```

Author: ```suggestion
2. Run TOPAS for each Control Point inside each beam, since we need the dose deposited by all the different energy layers present in the plan. 
3. Use convertBINtoRTDOSE.py to convert all of the different .bin files generated for each Control Point and save them inside the Patient folder as "RTDose_TOPAS_Beam{beamnumber}".
```

Author: ```suggestion
A "virtual" organ will be generated with the labelmaps given. A polinomial calibration will be done in order to adjust to this "virtual organ". Afterwards, this calibartion will be performed over the individual organs. You are also given the option to use the organ "CTV High" as reference for the calibration instead of this virtual one.
```

         --ParticlesperHistory < 1
         --NP=False
```

> 3) After this calibration is fully performed (see CalibrationMU2NP workflow), a txt file is generated containing ernergy/calibration factor dependence. 

Reviewer: ```suggestion
3) After this calibration is fully performed (see CalibrationMU2NP workflow), a txt file is generated containing energy-dependent calibration factor. 
```

           --NP=True
           --PPH=1e5 (for example)
```
### CalibrationMU2NP Workflow

> 1.  Run the script with --DVHCalibrated=False, as the optimization step must be performed first

Reviewer: ```suggestion
1.  Run the script with --DVHCalibrated=False, as the optimization step must be performed first
     Note that the script has been only tested with Patient1.
```

Reviewer: O haz una prueba con Patient2 para ver que salga lo mismo o parecido.

¿Todos los 20 pacientes tienen una contour llamada "CTV High"?

Author: si, miré en los dvh que sacamos al principio del todo y todos tienen mas o menos los mismos organos pero el ctv sí está siempre

2.  Optionally, set virtualOrgan=True to perform the calibration over the sum of all structures. By default, the calibration is performed using the CTV High.
3.  Outputs:
    - Calibration curve and calibration file.
    - Optimization curve together with the selected reference (Virtual Organ or CTV High).
    - Three plots (one per beam) showing the dose adjustment to the CTV High using the obtained calibration.
    - One plot comparing the calibrated sum of the three beams with the DICOM reference.

> 4. After running writeTROTStoDICOM with -MU2NPcalibrationFile and performing the TOPAS simulations with a large number of PPH, obtaining the calibrated RTDOSE files with convertBIN2RTDOSE, set --DVHCalibrated=True.

Reviewer: ```suggestion
4. After running writeTROTStoDICOM with -MU2NPcalibrationFile and performing the TOPAS simulations with a large number of PPH eg 1e5, obtain the calibrated RTDOSE files with convertBIN2RTDOSE, and rerun current script with flag --DVHCalibrated=True.
```
